js-sdk/integrations/react/translating.mdx

```diff
@@ -44,7 +46,7 @@ import { T } from "@tolgee/react";
 
 ...
 
-<T keyName="translation_key" />
+<T keyName="translation_key" defaultValue="Default value" params={...} />
```

Comment:
I don't think this is enough. As an user I would like to really see an example of the usage. How are you passing the params? Is it an object, an array, a single string? It is not clear This part is crucial and should be documented with care.